Rubberduck.VBEEditor/ComManagement/TypeLibs/TypeLibs.cs

```diff
@@ -534,5 +534,7 @@ public TypeLibWrapper Get(string searchLibName)
             }
             return retVal;
         }
+
+        protected override void Dispose(bool disposing) => base.Dispose(disposing);
```

Comment:
Why expression body here but not in other places?

Reply:
I was trying to be as consistent as I could - I only used the expression body for the overrides that did nothing but call `base.Dispose`.